library/std/src/panicking.rs

```diff
@@ -51,7 +51,7 @@ extern "Rust" {
     /// `payload` is passed through another layer of raw pointers as `&mut dyn Trait` is not
```

Comment:
This comment seems stale since its just `&mut dyn Trait` now. The `improper_ctypes` lint also seems unneeded now.

Reply:
Updated.